wallet/btc/wallet.go

```diff
@@ -0,0 +1,11 @@
+package btc
+
+import (
+	"github.com/DE-labtory/zulu/keychain"
+)
+
+type Wallet struct {
```

Comment:
**thought**: Duplicated name `wallet/btc/wallet` is on my mind. I think `wallet` package (not `wallet/btc/wallet`) would be larger and more abstract. And it would be changed to other name.